client/strings/de.json

```diff
@@ -20,7 +20,7 @@
   "ButtonCreate": "Erstellen",
   "ButtonCreateBackup": "Sicherung erstellen",
   "ButtonDelete": "Löschen",
-  "ButtonDownloadQueue": "Queue",
+  "ButtonDownloadQueue": "Download Warteschlange",
```

Comment:
Does `Warteschlange` translate okay on its own? This is displayed in the side rail so shorter text is preferred.

![image](https://user-images.githubusercontent.com/67830747/232830772-d87adf15-6b2b-4871-8de7-67a7280717c0.png)